### PR TITLE
Update contributing.adoc

### PR DESCRIPTION
**Pull request by matthewhelmke**
Just fixed a link that went stale when the docs moved from their own repo into the main one.

Oh, and hello!! :D

### CODE REVIEW

**Comment by sschu** [comment]
LGTM.

**Review by jonkoops — DISMISSED** [review]

File: docs/documentation/internal_resources/contributing.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 This guide will help new contributors use `git` and GitHub for Keycloak documentation submissions and suggestions.
 
-Before you start, please take a look at our writing templates contained in the link:https://github.com/keycloak/keycloak-documentation/tree/main/internal_resources[internal resources directory], which also describes each template file with information about when to use which. We are working to update our documentation to put procedural content, conceptual content, and reference content into separate .adoc files, and we kindly request that new contributions please use these templates.
+Before you start, please take a look at our writing templates contained in the link:https://github.com/keycloak/keycloak/tree/main/docs/documentation/internal_resources[internal resources directory], which also describes each template file with information about when to use which. We are working to update our documentation to put procedural content, conceptual content, and reference content into separate .adoc files, and we kindly request that new contributions please use these templates.
```
Comment (andymunro) [inline docs/documentation/internal_resources/contributing.adoc:6]:
```suggestion
Before you start,  check out our writing templates, which are contained in the link:https://github.com/keycloak/keycloak/tree/main/docs/documentation/internal_resources[internal resources directory], This directory also describes each template file and when to use that template. We are working to update our documentation to put procedural content, conceptual content, and reference content into separate .adoc files, and we request that new contributions use these templates.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Just a couple of changes to suggest @jonkoops

File: docs/documentation/internal_resources/contributing.adoc (hunk 1 above)
Comment (andymunro) [reply docs/documentation/internal_resources/contributing.adoc]: Thanks, @matthewhelmke. Welcome back to Keycloak documentation.

Comment (matthewhelmke) [reply docs/documentation/internal_resources/contributing.adoc]: Thanks for the warm welcome, @andymunro! :)

**Review by jonkoops — APPROVED** [review]
LGTM

**Comment by jonkoops** [comment]
Thanks for your contribution @matthewhelmke! @ssilvert could you merge this one in?

**Review by pedroigor — APPROVED** [review]
@matthewhelmke Glad to see you around and contributing to Keycloak :)

**Pull request merged**